src/TouringPanel.ts

```diff
@@ -63,9 +63,18 @@ class TouringPanel {
 
   private addEventListeners() {
     this.events.on(PanelTabEvents.SHOW_PANEL, () => {
+      if (this.active === true) {
+        return; //Don't update tasks when clicking on open touring button and touring panel is already open
+
+      }
+      if (this.active === false) {
+        this.updateOutput(true); //update tasks when panel opens
+      }
```

Comment:
```suggestion
      if (this.active === true) {
        return; // do not update tasks when clicking on open touring button and touring panel is already open
      }

      this.updateOutput(true); // update tasks when panel opens
```

You do not need the `if` here, because you exit the function when `active === true`.